applications/nrf5340_audio/unicast_client/main.c

```diff
@@ -420,18 +421,17 @@ void streamctrl_send(void const *const data, size_t size, uint8_t num_ch)
 	}
 }
 
-int streamctrl_start(void)
+int main(void)
 {
 	int ret;
-	static bool started;
 
-	if (started) {
-		LOG_WRN("Streamctrl already started");
-		return -EALREADY;
-	}
+	LOG_DBG("nRF5340 APP core started");
```

Comment:
Has the APP core started already, should this be "starting" or lower down before return?
Could this be an LOG_INF() so as to inform the user that it has competed the APP start-up?

Reply:
This is the first message after kernel boot, so it is only for debugging. Could potentially remove this as well, it's usually quite obvious if it doesn't get to this point